src/lightning-address.ts

```diff
@@ -104,6 +107,17 @@ export default class LightningAddress {
     return this.generateInvoice(callbackUrl);
   }
 
+  async boost(boost: Boost, amount: number = 0) {
+    const { destination, customKey, customValue } = parseKeysendResponse(this.keysendData);
```

Comment:
if we want to validate the keysend resonse. shouldn't we do that maybe better in the `fetch` call?